app/simplified-account-routes.js

```diff
@@ -23,7 +23,9 @@ simplifiedAccount.post(paths.simplifiedAccount.settings.serviceName.removeCy, en
 simplifiedAccount.post(paths.simplifiedAccount.settings.serviceName.edit, enforceLiveAccountOnly, permission('service-name:update'), serviceSettingsController.serviceName.postEditServiceName)
 
 // email notifications
-simplifiedAccount.get(paths.simplifiedAccount.settings.emailNotifications.index, permission('transactions:read'), serviceSettingsController.emailNotifications.get)
+simplifiedAccount.get(paths.simplifiedAccount.settings.emailNotifications.index, permission('transactions:read'), serviceSettingsController.emailNotifications.getEmailNotificationsSettingsPage)
+simplifiedAccount.get(paths.simplifiedAccount.settings.emailNotifications.collectionSettings, permission('email-notification-template:read'), serviceSettingsController.emailNotifications.getEmailCollectionPage)
+simplifiedAccount.post(paths.simplifiedAccount.settings.emailNotifications.collectionSettings, permission('email-notification-template:read'), serviceSettingsController.emailNotifications.postEditEmailCollection)
```

Comment:
```suggestion
simplifiedAccount.get(paths.simplifiedAccount.settings.emailNotifications.index, permission('transactions:read'), serviceSettingsController.emailNotifications.get)
simplifiedAccount.get(paths.simplifiedAccount.settings.emailNotifications.collectionSettings, permission('email-notification-template:read'), serviceSettingsController.emailNotifications.getEmailCollectionMode)
simplifiedAccount.post(paths.simplifiedAccount.settings.emailNotifications.collectionSettings, permission('email-notification-template:read'), serviceSettingsController.emailNotifications.postEmailCollectionMode)
```

make the appropriate changes in the controller as well

alternatively, have a separate controller for email collection mode and require it as an export in the email notifications controller. e.g. `serviceSettingsController.emailNotifications.emailCollectionMode.get`. This might be sensible given there are a few sub settings and will avoid a very large controller

Reply:
I did wonder about having separate controllers for each email notification setting but I thought it might get more confusing to have several controllers each with just two methods in. Not sure either way on it though

Reply:
I think the email notifications functionality is simple enough we don't need multiple controllers. I also think having multiple controllers is a good shout to keep them simple. I'm happy to stick the payment confirmation emails and refund emails functionality in different controllers but in the interests of time I think we should keep this controller as is. Moving this out is going to mean refactoring of the test class as well and I don't think it's worth it given the relative simplicity of the email notifications functionality just now.

Reply:
I also think that with the email notifications functionality it's only using the same connector API, so it's not complex and not hard to test at all. I can see the case for using different controllers when different methods/functionalities use different external APIs which would make understand and testing easier.

Reply:
That said I'm happy to make this refactor in a later PR. Would like to get the edit collection mode functionality in just now.